minio/go.mod

```diff
@@ -0,0 +1,34 @@
+module main
+
+go 1.22
+
+require (
+	github.com/gofiber/fiber/v2 v2.52.5
+	github.com/minio/minio-go/v7 v7.0.80
```

Comment:
We already have an official Min.io client that would reduce this recipe boilerplate a lot. Why not use that?

https://github.com/gofiber/storage/tree/main/minio

Reply:
You can access the underlaying client by using `s.Conn()`

Reply:
Yes, you are correct—I've replaced the MinIO client with Fiber's official MinIO storage to reduce the boilerplate. Thanks for the suggestion!